sca-wrapper-play-30/src/main/scala/uk/gov/hmrc/sca/services/WrapperService.scala

```diff
@@ -63,7 +64,8 @@ class WrapperService @Inject() (
     optTrustedHelper: Option[TrustedHelper] = None,
     fullWidth: Boolean = true,
     hideMenuBar: Boolean = false,
-    disableSessionExpired: Boolean = appConfig.disableSessionExpired
+    disableSessionExpired: Boolean = appConfig.disableSessionExpired,
+    bespokeUserResearchBannerConfig: Option[BespokeUserResearchBannerConfig] = None
```

Comment:
This requires a code change to update the banner.

Use the single-account-wrapper-data to push the banner to the service.

See: https://github.com/hmrc/app-config-base/blob/198ca49963d7a5be6f8d83084f84d9827d4c6cc1/single-customer-account-wrapper-data.conf#L53